### PR TITLE
Ap images feature

### PR DESCRIPTION
**Pull request by syed-imam**
Renamed the social media component into explore component. Added the dropdown options for both Twitter and AP in the search bar

### CODE REVIEW

File: src/app/views/explore/explore.module.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { SocialMediaComponent } from './social-media/social-media.component';
+import { ExploreComponent } from './explore/explore.component';
 import {RouterModule} from "@angular/router";
-import {SocialMediaRoutes} from "./social-media.routing";
+import {ExploreRoutes} from "./explore.routing";
```
Comment (rosscooperman) [inline src/app/views/explore/explore.module.ts]: `{ ... }` or `{...}`? I think our standard is `{ ... }` but either is fine if it's consistent 😄

File: src/app/views/explore/explore.module.ts [hunk 2]
```diff
@@ -37,6 +37,7 @@ import {
         MatInputModule,
         MatProgressBarModule,
         FlexLayoutModule,
+        ReactiveFormsModule,
     ]
 })
-export class SocialMediaModule { }
+export class ExploreModule { }
```
Comment (rosscooperman) [inline src/app/views/explore/explore.module.ts]: What is the empty `{ }` export? Is that an Angular thing?

File: src/app/views/explore/explore.routing.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Routes } from '@angular/router';
+import {ExploreComponent} from "./explore/explore.component";
```
Comment (rosscooperman) [inline src/app/views/explore/explore.routing.ts]: You may want to add a linter (`prettier`) at some point so it keeps you honest re: ' versus ", etc. We use it on most of our TypeScript repos and it forces us to be consistent.

File: src/app/views/explore/explore.module.ts (hunk 2 above)
Comment (syed-imam) [reply src/app/views/explore/explore.module.ts]: I believe it is the Angular syntax to expose modules

Screen Shot 2020-02-06 at 4 08 28 PM

Comment (syed-imam) [reply src/app/views/explore/explore.module.ts]: A module is just collection of components in Angular

Comment (rosscooperman) [reply src/app/views/explore/explore.module.ts]: Gotcha, interesting. Had never seen that syntax before.

**Comment by kspector** [comment]
@syed-imam is this ready for rereview?

File: src/app/views/explore/explore.module.ts (hunk 1 above)
Comment (syed-imam) [reply src/app/views/explore/explore.module.ts]: Import spacing is made consistent in latest commit

**Pull request merged**